homeassistant/components/yale_smart_alarm/button.py

```diff
@@ -57,6 +58,16 @@ async def async_press(self) -> None:
         if TYPE_CHECKING:
             assert self.coordinator.yale, "Connection to API is missing"
 
-        await self.hass.async_add_executor_job(
-            self.coordinator.yale.trigger_panic_button
-        )
+        try:
+            await self.hass.async_add_executor_job(
+                self.coordinator.yale.trigger_panic_button
+            )
+        except YALE_ALL_ERRORS as error:
+            raise HomeAssistantError(
+                translation_domain=DOMAIN,
+                translation_key="trigger_panic",
```

Comment:
To be consistent with the exception, would this not be better named as `could_not_trigger_panic`?

Reply:
Thanks! 👍